### ISSUE TITLE
AndroidTV user interface overhaul + some related features

### ISSUE THREAD

**Issue by Feuerswut** [TeamNewPipe/NewPipe#5789]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
(this is kinda related to some requests/bugs)
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.
(Everything is related)

#### Describe the feature you want
Better navigation on Android TV using a design *like* in the YouTube for Android TV App.
This has potential to improve usability A LOT

#### Is your feature request related to a problem? Please describe it
- You have to press "OK" way to often, so it feels like using a windows 10 computer with only a keyboard (possible, but annoying)
-> Goal should be least button inputs to reach a video
-> My suggestions are below
-> Capitalize on all directions, e.g. left/right/up/down on the remote, dont use OK too often (only if needed; e.g. confirm search, confirm fullscreen; *NOT "open settings" or "open this tab"*
-> *Inputs are more of a hassle then the availability of screen space to display options/tabs*

- You cannot change the home screen unless you import settings from mobile (which is... annoying)
- You cannot easily import history and subscriptions
(Maybe you could do a temporary local server + QR-Code
- You cannot update the app easily: Most vendors do not have a browser that supports downloads and/or a file app that supports installing/updating Apps
-> The app should have the ability to update itself in the "Update" Tab in the settings.
- Skipping forward obstructs making fullscreen (you have to press DOWN on the remote to get to this option; NOT intuitive)

#### Additional context
- You have to swipe to change/rearrange/remove the tabs in the "home" menu.
- My suggestion is the removal of the whole home menu on Android TV using a toggle in settings.
- Instead, you navigate in the now default "Burger" (three lines) menu and what would be displayed after pressing "OK" should be displayed instantaneously (if you hover over it) and can be accessed by pressing RIGHT on the remote
- Visual space is not an issue on Android TV (unlike on mobile) so this design is already implemented by the YouTube for Android TV app, refer to it to see what I mean (you do NOT press OK very often.
- Search then would also have to be moved below the "New Pipe" Logo in the Burger Menu
- The "New Pipe"-Logo atop the Burger Menu should also instantly display change options for SoundCloud etc.

#### How will you/everyone benefit from this feature?
This feature will help us colonize the Android TV menuing
Dankeschön! (;

**Comment by peat80** [comment]
Great suggestions. 😍
AndroidTV - UI could really need some love. 🤔👍

Maybe we could open a bounty or something and put some cash in there to attract developers to work on improvements for ATV?!? 💶
Would be willing to spent some bucks for something like this. 😀

**Comment by PhillyMay** [comment]
Another nice feature would be an "Update Feed on App Startup" feature. 
Currently updating a feed is a bit frustrating because its somewhat complicated. You have to know, that you can only navigate to the update button by clicking "up" on your remote when you are on the center Video thumbnail. If you are on the top left Video thumbnail, it brings you to the top navigation bar

**Comment by DraconicNEO** [comment]
Maybe a good idea would be to add a TV Ui similar to how VLC player does it and have it be able to switch automatically depending on the device but also be toggleable in the settings.

**Comment by ingmarcosp** [comment]
I'm using android tv 9 and moving arround with the remote control is quite difficult, even changing to tablet view, for example to got ot fullscreen you have to move to the 2nd video con the right list, the move left to rise the fullscreen button. Some time you get stock in the list of video. Maybe a simple tv UI it will be greate!

**Comment by peat80** [comment]
@ingmarcosp to reach fullscreen you just have to open the osd on the video, then press 2x down and then enter.

**Comment by opusforlife2** [comment]
@peat80 What is "osd"?

**Comment by DraconicNEO** [comment]
> @peat80 What is "osd"?

I think he meant On screen Display, although I've never heard anyone use OSD to refer to it before.

**Comment by peat80** [comment]
Yes, i meant on screen display. With other words click in the video, then the you see the big play/pause button (which i call osd) then 2x down should focus the fullscreen button and enter then opens fullscreen.

**Comment by mhmdanas** [comment]
> I think he meant On screen Display, although I've never heard anyone use OSD to refer to it before.

The term "OSD" is used by [MPV](https://mpv.io), so they might have heard the term from that project.
